From a4fb8e769bbd4438ce722fe7c2d46eb2cf822789 Mon Sep 17 00:00:00 2001
From: Calum Lind <calumlind+deluge@gmail.com>
Date: Mon, 4 Aug 2014 18:53:54 +0100
Subject: [PATCH 07/24] [#2470] [Console] Fix console parsing args

This negates the need for quoting a single command with an arg e.g.
    deluge-console del --remove_data torrrent_file

Multiple commands separated by semi-colon still require quoting.
---
 deluge/ui/console/main.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/deluge/ui/console/main.py b/deluge/ui/console/main.py
index d700b83..a00eca3 100644
--- a/deluge/ui/console/main.py
+++ b/deluge/ui/console/main.py
@@ -64,6 +64,7 @@ def __init__(self):
         group = optparse.OptionGroup(self.parser, "Console Commands",
             "\n".join(cmds.keys()))
         self.parser.add_option_group(group)
+        self.parser.disable_interspersed_args()
 
     def start(self):
         super(Console, self).start()
-- 
2.1.0

